extensions/roc-package-web-app-react-dev/src/config/roc.config.js

```diff
@@ -19,6 +19,10 @@ export default {
                 locales: [],
             },
 
+            reactRouterScrollAsync: {
+                enabled: true,
```

Comment:
What do you think about changing this into `build.disableReactRouterScrollAsync`? An object seems a little overkill here.

Reply:
👍

Reply:
Maybe that `disableReactRouterScrollAsync`, `routes` and `useDefaultRoutes` should be moved under a `reactRouter` key, similar to how it's done for redux? (in a later PR)